baselines/acktr/kfac.py

```diff
@@ -1,16 +1,45 @@
-import tensorflow as tf
-import numpy as np
 import re
-from baselines.acktr.kfac_utils import *
 from functools import reduce
 
+import tensorflow as tf
+import numpy as np
+
+from baselines.acktr.kfac_utils import detect_min_val, factor_reshape, gmatmul
+
 KFAC_OPS = ['MatMul', 'Conv2D', 'BiasAdd']
 KFAC_DEBUG = False
 
 
-class KfacOptimizer():
-
-    def __init__(self, learning_rate=0.01, momentum=0.9, clip_kl=0.01, kfac_update=2, stats_accum_iter=60, full_stats_init=False, cold_iter=100, cold_lr=None, async=False, async_stats=False, epsilon=1e-2, stats_decay=0.95, blockdiag_bias=False, channel_fac=False, factored_damping=False, approxT2=False, use_float64=False, weight_decay_dict={},max_grad_norm=0.5):
+# TODO: fix missing/inacurate comments
```

Comment:
@kalifou What is missing ? blockdiag_bias ?

Reply:
This was from me, I was not sure if the documentation is accurate or not